_source/user-guide/infrastructure-monitoring/jenkins.md

```diff
@@ -0,0 +1,42 @@
+| Jenkins Health              | jenkins\_health\_check\_score\_gauge                                                                                                                                                                       | The ratio of health checks reporting success to the total number of health checks.                                                                                                                                                                                                                                                                                          |
+| Total Jenkins Nodes         | jenkins\_node\_count\_value\_gauge                                                                                                                                                                         | The number of build nodes available to Jenkins, both on-line and off-line.                                                                                                                                                                                                                                                                                                  |
+| Online Nodes                | jenkins\_node\_online\_value\_gauge                                                                                                                                                                        | The number of build nodes available to Jenkins and currently on-line.                                                                                                                                                                                                                                                                                                       |
+| Offline Nodes | jenkins\_node\_offline\_value\_gauge                                                                                                                                                                       | The number of build nodes available to Jenkins but currently off-line.                                                                                                                                                                                                                                                                                                      |
+| Enabled Projects | jenkins\_project\_enabled\_count\_value\_gauge  | Enabled projects of Freestyle type |
```

Comment:
Doesn't have to be of Freestyle type (FreeStyle is the most common one, but there are other types as well).
You can leave it just as 'Enabled projects' - Jenkins users knows what it means